Doc/using/configure.rst

```diff
@@ -22,6 +22,8 @@ Features required to build CPython:
 
 * On Windows, Microsoft Visual Studio 2017 or later is required.
 
+* Sphinx 6.2 or newer is required to build the Python documentation.
```

Comment:
This seems a bit out of place here to me, but I don't have a better suggestion for where to put it off the top of my head.  Do we actually mention building the documentation elsewhere in the docs?  I can't find one with a short search.

Reply:
Yeah, I agree—it seems quite out of place here. The meta-documentation for the docs themselves [lives in the devguide](https://devguide.python.org/documentation/), and this mention should presumably go under [the appropriate section](https://devguide.python.org/documentation/start-documenting/#building-the-documentation).

Reply:
I'd prefer to avoid mentioning the Sphinx version in the devguide and needing to keep it in sync. It's defined in the `requirements.txt` so whatever is needed will be installed as required. Plus it's different for older branches.

Reply:
Are you saying that the whole section "Build Requirements" is irrelevant in Python documentation. Or that mentioning the minimum Sphinx version in "Build Requirements" is irrelevant?

For me, it's convenient to have the same repository where we actually specify ``needs_sphinx`` (Doc/conf.py) and we clearly document that version. Having two Git repositories just make it less pratical.

> Do we actually mention building the documentation elsewhere in the docs?

I added this section recently. I wrote "Sphinx 6.2 or newer is required **to build the Python documentation**." If someone doesn't care about the doc, this sentence can be ignored.

Reply:
> I'd prefer to avoid mentioning the Sphinx version in the devguide and needing to keep it in sync. It's defined in the requirements.txt so whatever is needed will be installed as required. Plus it's different for older branches.

FWIW, on second thought I agree, and in fact it was I that removed the references to such recently when overhauling the section I linked. We could mention the files the requirements are defined in, though, and directly link them with the `:cpy-file:` role. That would allow users to easily check the versions for their desired branch without having to duplicate them multiple places, which will inevitably get out of sync.